tests/hpc/changelog

```diff
@@ -5,4 +5,5 @@ All notable changes to this project will be documented in this file
 ### Added
 Add the change log along with the README
 ### Changed
+Move libsolv test to the HPC group as this should be well crafted for the HPC use
```

Comment:
I think the idea of the changelog is that you would create a new "version" and mention your changes there

Reply:
depends , I think in this certain case , it is good enough without dive into versions for HPC tests . 
@schlad :+1:  I like the idea

Reply:
> depends , I think in this certain case , it is good enough without dive into versions for HPC tests .

I fully agree. +2